### PR TITLE
[TVMScript][Parser] Add more warp-level builtins and `Range`

### PR DESCRIPTION
**Pull request by yzh119**
# Motivation
Several builtins "tvm_storage_sync", "tvm_warp_shuffle", "tvm_warp_shuffle_up", "tvm_warp_shuffle_down", "tvm_warp_activemask" and `Range` will appear in TVMScript printer but are missing in TVMScript parser. This PR fix the behavior.
 
@cyx-6 @junrushao

### CODE REVIEW

File: python/tvm/script/ir_builder/tir/ir.py [hunk 1]
```diff
@@ -1782,6 +1798,11 @@ def wrapped(*args, **kwargs):
 tvm_bmma_sync = _op_wrapper(_tir_op.tvm_bmma_sync)
 tvm_fill_fragment = _op_wrapper(_tir_op.tvm_fill_fragment)
 tvm_store_matrix_sync = _op_wrapper(_tir_op.tvm_store_matrix_sync)
+tvm_storage_sync = _op_wrapper(_tir_op.tvm_storage_sync)
```
Comment (junrushao) [inline python/tvm/script/ir_builder/tir/ir.py:1801]: No need to use op_wrapper - this is only used for backward compatibility

**Review by junrushao — APPROVED** [review]
Otherwise LGTM

File: python/tvm/script/ir_builder/tir/ir.py (hunk 1 above)
Comment (yzh119) [reply python/tvm/script/ir_builder/tir/ir.py]: removed.

**Review by junrushao — APPROVED** [review]
LGTM

**Comment by junrushao** [comment]
Very cool! Thanks for the PR

**Pull request merged**